server/resources/translations/common.es_us.json

```diff
@@ -0,0 +1,18 @@
+{
+  "alertAcknowledged": "Otro personal de auxilio ha confirmado esta solicitud. (No necesita responder a este mensaje).",
+  "alertAdditionalAlert": "Se generó una alerta {{ alertTypeDisplayName }} adicional en {{ deviceDisplayName }}",
+  "alertCompleted": "Gracias. Esta sesión ya está completa. (No necesita responder a este mensaje).",
+  "alertFallback": "No se respondió a una alerta de verificación de {{ deviceDisplayName }}. Por favor, verifíquelo.",
+  "alertReminder": "Este es un recordatorio para que verifique {{ deviceDisplayName }}",
+  "alertStart": "Esta es una alerta {{ alertTypeDisplayName }}. Verifique en {{ deviceDisplayName }}. Responda con 'ok' una vez que lo haya verificado.",
+  "errorNoSession": "Error: No se encontró ninguna sesión activa",
+  "incidentCategorized": "El incidente se clasificó como {{ incidentCategory }}. Gracias. Esta sesión ya está completa. (No necesita responder a este mensaje).",
```

Comment:
Newlines have to be added to this file explicitly using `\n`. So, when there is a blank line between the incident category and "Gracias", I was expecting to see `\n\n` similar to how it is in the English file: https://github.com/bravetechnologycoop/BraveSensor/blob/16381dd4e0db93ff1b26b7916394d1c79962507f/server/resources/translations/common.en.json#L9